### ISSUE TITLE
`ResponseBody` Test Comparison bug

### ISSUE THREAD

**Issue by Lakshmikanth2001** [rangav/thunder-client-support#878]
**Describe the bug**
Error in `Response Body` test result 

**To Reproduce**

![Screenshot 2022-11-16 at 3 27 27 PM](https://user-images.githubusercontent.com/52835045/202149292-0310610d-a955-4c27-b474-41b85900807e.png)

**Expected behavior**

```
Response Body equal to { "a": 3, "b" : 2 } => Actual: { "a": 3, "b": 2 }  Pass
```

**Platform:** 
 - OS: `MAC OS Monterey`
 - vscode version: `1.73.1`
 - extension version: `v1.20.1`

**Solution:**

Compare `Response Body` just as a string

**Comment by rangav** [comment]
It is difficult to correctly match json strings, there will be white spaces and new lines.

You should `Json Query` in Tests tab

<img width="1400" alt="Screenshot 2022-11-16 at 15 07 56" src="https://user-images.githubusercontent.com/8637550/202217981-0d9fd0d6-14f4-4e4a-acc8-27a8624384b9.png">

**Comment by Lakshmikanth2001** [comment]
so `responseBody` option will be of less use right?

we can use `JSON.parse(targetString)` that will handle all additional white spaces and commas

**Comment by rangav** [comment]
ResponseBody is useful check `IsJson` or not and useful to check `contains` operation. 

Not useful for `json` comparison, but useful text response results.

I will research in the meanwhile if there is any better approach to compare json.

**Comment by Lakshmikanth2001** [comment]
Thanks for your time, if this feature comes that will be a great addition because we have pre populated `JSON` objects we can directly copy it in `responseBody`

**Comment by rangav** [comment]
Thanks for feedback, will look into it

**Comment by rangav** [comment]
found a way to compare json with response, the solution is will part of next update, which planned to release tomorrow or Friday.

**Comment by rangav** [comment]
This Bug is fixed is now and published to marketplace, please update to v2.0.0

See all features released
https://github.com/rangav/thunder-client-support/releases/tag/v2.0.0

Please let me know your feedback.

Save the json text in `environment variable` and use `stringify` filter as below

<img width="1186" alt="Screenshot 2022-11-17 at 12 57 36" src="https://user-images.githubusercontent.com/8637550/202452629-791e793d-bbcb-4a6f-ab79-a83b49458cf8.png">
